src/components/PantallaValidacion/RedButton/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Styles from './index.module.scss';
```

Comment:
Intenta que los imports que se usen en el código no empiecen con mayúscula así no se los confunde con otro componente.
```suggestion
import styles from './index.module.scss';
```